src/protocolsupport/protocol/typeremapper/pe/PEDataValues.java

```diff
@@ -406,21 +406,23 @@ public static PEEntityData getEntityData(NetworkEntityType type) {
 	}
 
 	public static class PEEntityData {
-		private BoundingBox boundingBox;
-		private Offset offset;
-		private RiderInfo riderInfo;
+		// Note that these fields must have non-standard naming to match
+		// the format in entitydata.json
+		private BoundingBox BoundingBox;
```

Comment:
Please use Gson's SerializedName annotation on every field, instead of changing the field name, to keep consistency between classes

`@SerializedName("BoundingBox")`
`private BoundingBox boundingBox`

etc etc etc